bin/make-tag.py

```diff
@@ -5,17 +5,30 @@
 
 import datetime
 import subprocess
+import sys
 
 
 def _check_output(*args, **kwargs):
     return subprocess.check_output(*args, **kwargs).decode("utf-8").strip()
 
 
+def get_remote_name():
+    # Figure out remote to push tag to
+    remote_output = _check_output("git remote -v".split())
+
+    for line in remote_output.splitlines():
+        line = line.split("\t")
+        if "mozilla-services" in line[1]:
+            return line[0]
+
+    raise Exception("Can't figure out remote name for mozilla-services.")
```

Comment:
This susses out the remote name so we can all have different remote names and we're fine.